src/osx/listbox_osx.cpp

```diff
@@ -131,10 +133,10 @@ void wxListBox::FreeData()
 
 void wxListBox::DoSetFirstItem(int n)
 {
-    // osx actually only has an implementation for ensuring the visibility of a row, it does so  
+    // osx actually only has an implementation for ensuring the visibility of a row, it does so
```

Comment:
Please configure your editor to avoid automatically stripping whitespace. TIA!

Reply:
BTW, the .editorconfig file in the wxWidgets project has `trim_trailing_whitespace` enabled which is how this happened.

Reply:
Hmm, we do want to avoid whitespaces in the new code. But we don't want to mix up removing them with the other, significant, code changes. The solution is probably to trim all the existing whitespace (I think it's only present in Mac sources...).